rust/Cargo.toml

```diff
@@ -8,7 +8,6 @@ description = "Cross Platform Library for providing To Do List data"
 
 [lib]
 name = "toodle"
-crate-type = ["staticlib", "cdylib"]
```

Comment:
I'm not sure how to make this iOS and Android-only...

Reply:
`[target.'cfg(target_os = "ios")']` would be used for iOS, `[target.'cfg(target_os="android")']` for Android. It looks like you can string flags together, so this should work:

```
 [lib]
 name = "toodle"
 [target.'cfg(target_os="android", target_os="ios")'.lib]
 crate-type = ["staticlib", "cdylib"]
```

Reply:
This doesn't work for me, unfortunately. 😞

```toml
[lib]
name = "toodle"
[target.'cfg(any(target_os="android", target_os="ios"))'.lib]
crate-type = ["staticlib", "cdylib"]
```

I get `warning: unused manifest key: target.cfg(any(target_os="android", target_os="ios")).lib`. Same with just `[target.'cfg(target_os="ios"))'.lib]`.

In both cases, `cargo lipo` fails with `Failed to parse 'cargo read-manifest' output: No target with kind 'staticlib' found`.